frontend/src/components/filters/Filters.tsx

```diff
@@ -133,7 +134,16 @@ export default function Filters({ filters, onFilterChange }: Props) {
     const chain = useSelector((state: RootState) => state.chain.chain)
     const account = useSelector((state: RootState) => state.accountInfo.account);
 
-    const destinationChains = useMemo(() => Object.values(ChainEnum).filter(x => x !== chain), [chain])
```

Comment:
maybe in some utils or constants file, create:
skylineChains: []ChainEnum
reactorChains: []ChainEnum

then you can import the right one, and keep this logic:
const destinationChains = useMemo(() => chains.filter(x => x !== chain), [chain])